activerecord/lib/arel/table.rb

```diff
@@ -78,6 +78,10 @@ def having(expr)
       from.having expr
     end
 
+    def comment(*cmt)
```

Comment:
There is no reason to add the unused internal API.

Reply:
This was added because some developers do use `Arel` directly in applications. It has been removed. :+1:

Reply:
Still remained.